bootique-cayenne-jcache/pom.xml

```diff
@@ -101,6 +106,38 @@
         </dependency>
     </dependencies>
 
+    <dependencyManagement>
```

Comment:
"cayenne-server" and "cayenne-java8" should probably not be here. They come as transitive dependencies from "bootique-cayenne"